rero_ils/modules/items/api/record.py

```diff
@@ -257,3 +257,18 @@ def get_owning_pickup_location_pid(self):
         """Returns the pickup location for the item owning location."""
         library = self.get_library()
         return library.get_pickup_location_pid()
+
+    @property
+    def notes(self):
+        """Return notes related to this item."""
+        return self.get('notes', [])
+
+    def get_note(self, note_type):
+        """Return the a specific note related to this item.
+
+        :param note_type: the type of note (see ``ItemNoteTypes``)
+        :return the content of the note, None if note type isnot found
+        """
+        notes = [note.get('content') for note in self.notes
+                 if note.get('type') == note_type]
+        return next(iter(notes), None)
```

Comment:
`None` does it mean, a note with the given type exists and it has the value None. Or  the note of the given  type does  not exist.

Reply:
The method return `None` if the desired note isn't present into item data. According to the jsonSchema you cannot have a note without content.